app/src/main/java/io/github/droidkaigi/confsched2018/presentation/contributor/ContributorsActivity.kt

```diff
@@ -35,6 +35,7 @@ class ContributorsActivity : BaseActivity(), HasSupportFragmentInjector {
         setSupportActionBar(binding.toolbar)
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
         navigationController.navigateToContributor()
+        drawerMenu.setup(binding.toolbar, binding.drawerLayout, binding.drawer)
     }
 
```

Comment:
Please add `onBackPressed()` for closing drawer :bow:
```
override fun onBackPressed() {
     if (drawerMenu.closeDrawerIfNeeded()) {
         super.onBackPressed()
     }
}
```
ref: https://github.com/DroidKaigi/conference-app-2018/pull/145

Reply:
Thanks for pointing out. I added that one!

Reply:
@KensukeIzumi Great! Thanks! (Excuse me for cutting in...)